<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inquiry</class>
 <widget class="QWidget" name="inquiry">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>544</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>搜索航班</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>921</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 255, 251);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>861</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QFrame {
	background-color: rgb(255, 255, 255);
	background-color: rgb(255, 255, 255);
        border: 1px solid #E0E0E0;
        border-radius: 10px;
 
    }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>出发地</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目的地</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>出发日期</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditArrival">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>142</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDepart">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchBtn">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{border-radius:4px;background:#409eff;color:white;font-size:14px}
QPushButton:hover {
         background-color: #ADD8E6;&quot;        /* 悬停时背景颜色 */
        }</string>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDate">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>167</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>210</y>
      <width>861</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);</string>
    </property>
    <column>
     <property name="text">
      <string>航班号</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>时间</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>经济舱票价</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>头等舱票价</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商务舱票价</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>经济舱人数</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>头等舱人数</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商务舱人数</string>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#HomeButton{
	image: url(:/Image/Imageres/user (1).png);
    background: transparent; /* 背景透明 */
    border: none; /* 移除边框 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ReturnBtn{
	image: url(:/Image/Imageres/back.png);
    background: transparent; /* 背景透明 */
    border: none; /* 移除边框 */
    cursor: pointing-hand; 
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Image/Imageres/BackGround4.jpg);
background-image: url(:/Imageres/airBg.jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>tableWidget</zorder>
   <zorder>HomeButton</zorder>
   <zorder>ReturnBtn</zorder>
   <zorder>frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
